zap/src/test/java/org/parosproxy/paros/network/HttpResponseHeaderUnitTest.java

```diff
@@ -126,4 +130,43 @@ public void shouldSetEmptyReasonPhraseIfNull() throws Exception {
         assertThat(header.getReasonPhrase(), is(equalTo("")));
         assertThat(header.getPrimeHeader(), is(equalTo("HTTP/1.1 200")));
     }
+
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
+                "text/html", // Text but not css
+                "image/png", // Not text or css
+                "teXt/hTmL", // Mixed case
+                "text/html; charset=UTF-8", // Expected charset
+                "text/html;charset=UTF-8", // Charset without space
+                "charset=UTF-8; text/html" // Charset first
+            })
+    public void isCssShouldReturnFalseWhenContentTypeIndicatesCss(String contentType) {
```

Comment:
..WhenContentTypeDoesnNotIndicateCss ?

Reply:
Oops, paste fail ...

Reply:
It was added one `n` before `Not`.

Reply:
:( we did it to each other, I copied psiinon's text above for the adjustment :(

Fixed the fix :)